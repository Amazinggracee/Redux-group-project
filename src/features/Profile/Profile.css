.profile {
  display: flex;
  justify-content: space-around;
}

.listtitle {
  margin-bottom: 9px;
  padding: 0;
  font-size: xx-large;
}

.listbox {
  display: flex;
  flex-direction: column;
  text-align: start;
  width: 45%;
  height: max-content;
}

.itemlist {
  all: unset;
  display: flex;
  flex-direction: column;
  justify-content: flex-end;
  border: 1px solid rgb(101, 101, 103);
}

.listitems {
  border: 1px solid rgb(101, 101, 103);
  list-style: none;
  font-size: larger;
  height: auto;
  padding: 20px;
}
