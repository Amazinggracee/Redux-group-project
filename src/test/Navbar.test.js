import React from 'react';
import renderer from 'react-test-renderer';
import Profile from '../features/Profile/Profile';
import Missions from '../features/Missions/Missions';
// import Rockets from '../features/Rockets/Rockets';

it('renders correctly', () => {
  const tree = renderer.create(<Profile />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly', () => {
  const tree = renderer.create(<Missions />).toJSON();
  expect(tree).toMatchSnapshot();
});

// it('renders correctly', () => {
//   const tree = renderer.create(<Rockets />).toJSON();
//   expect(tree).toMatchSnapshot();
// });
