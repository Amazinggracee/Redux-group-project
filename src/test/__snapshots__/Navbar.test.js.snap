// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="profile"
>
  <div
    className="listbox"
  >
    <h2
      className="listtitle"
    >
      My Missions
    </h2>
    <ul
      className="itemlist"
    >
      <li
        className="listitems"
      >
        mars
      </li>
      <li
        className="listitems"
      >
        Jupiter
      </li>
    </ul>
  </div>
  <div
    className="listbox"
  >
    <h2
      className="listtitle"
    >
      My Rockets
    </h2>
    <ul
      className="itemlist"
    >
      <li
        className="listitems"
      >
        Falcon 19
      </li>
    </ul>
  </div>
</div>
`;

exports[`renders correctly 2`] = `
<div
  className="missions"
>
  <table
    cellPadding="0"
    cellSpacing="0"
  >
    <thead>
      <tr>
        <th>
          Mission
        </th>
        <th>
          Description
        </th>
        <th>
          Status
        </th>
        <th>
           
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>
          mars
        </td>
        <td>
          dffffffffffffffffffffffffffff
        </td>
        <td />
        <td>
          <button
            type="button"
          >
            Join Mission
          </button>
        </td>
      </tr>
      <tr>
        <td>
          mars
        </td>
        <td>
          dffffffffffffffffffffffffffff
        </td>
        <td />
        <td>
          <button
            type="button"
          >
            Join Mission
          </button>
        </td>
      </tr>
      <tr>
        <td>
          mars
        </td>
        <td>
          dffffffffffffffffffffffffffff
        </td>
        <td />
        <td>
          <button
            type="button"
          >
            Join Mission
          </button>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;

exports[`renders correctly 3`] = `
<div
  className="missions"
>
  <table
    cellPadding="0"
    cellSpacing="0"
  >
    <thead>
      <tr>
        <th>
          Mission
        </th>
        <th>
          Description
        </th>
        <th>
          Status
        </th>
        <th>
           
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>
          mars
        </td>
        <td>
          dffffffffffffffffffffffffffff
        </td>
        <td />
        <td>
          <button
            type="button"
          >
            Join Mission
          </button>
        </td>
      </tr>
      <tr>
        <td>
          mars
        </td>
        <td>
          dffffffffffffffffffffffffffff
        </td>
        <td />
        <td>
          <button
            type="button"
          >
            Join Mission
          </button>
        </td>
      </tr>
      <tr>
        <td>
          mars
        </td>
        <td>
          dffffffffffffffffffffffffffff
        </td>
        <td />
        <td>
          <button
            type="button"
          >
            Join Mission
          </button>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
